### PR TITLE
txpool: fix typo

### PR DESCRIPTION
**Pull request by tcoratger**

### CODE REVIEW

File: crates/transaction-pool/src/lib.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@
 //! The pool itself does not validate incoming transactions, instead this should be provided by
 //! implementing `TransactionsValidator`. Only transactions that the validator returns as valid are
 //! included in the pool. It is assumed that transaction that are in the pool are either valid on
-//! the current state or could become valid after certain state changes. transaction that can never
+//! the current state or could become valid after certain state changes. Transaction that can never
```
Comment (Rjected) [inline crates/transaction-pool/src/lib.rs:38]:
```suggestion
//! the current state or could become valid after certain state changes. Transactions that can never
```

**Review by Rjected — CHANGES_REQUESTED** [review]
only one grammar suggestion

**Review by Rjected — APPROVED** [review]
lgtm, thanks!

File: crates/transaction-pool/src/lib.rs [hunk 2]
```diff
@@ -35,7 +35,7 @@
 //! The pool itself does not validate incoming transactions, instead this should be provided by
 //! implementing `TransactionsValidator`. Only transactions that the validator returns as valid are
 //! included in the pool. It is assumed that transaction that are in the pool are either valid on
```
Comment (tcoratger) [inline crates/transaction-pool/src/lib.rs:37]:
```suggestion
//! included in the pool. It is assumed that transactions that are in the pool are either valid on
```

**Pull request merged**